site/en/release/backward-compatibility.md

```diff
@@ -60,6 +60,4 @@ For every incompatible change, the issue specifies the following:
 * Description of the changed functionality
 * Migration recipe
 
-The incompatible change issue is closed when the incompatible flag is flipped at
-HEAD. All incompatible changes that are expected to happen in release X.Y
-are marked with a label "breaking-change-X.Y"."
+When an incompatible change is ready for migration with Bazel at HEAD (therefore, also with the next Bazel rolling release), it should be marked with the `migration-ready` label. The incompatible change issue is closed when the incompatible flag is flipped at HEAD.
```

Comment:
Migration ready is a signal that we are committed to the change and it will become permanent, but it is not necessarily in the next LTS release. We need to account for ones like the non-empty glob change, which has been migration ready for years, but we are still not ready to flip.

We should leave the bug open until we delete the code behind the flag, not just flip it. It is not over until it can't be reverted.

Reply:
Yes, `migration-ready` doesn't imply it will be flipped in the next major release, it totally depends on the migration status and other potential blockers. Having this label tells the incompatible flags pipeline which flags to test so that we can check the migration statuses for downstream projects.

> We should leave the bug open until we delete the code behind the flag, not just flip it. It is not over until it can't be reverted.

I agree with it, it's just there are already many issues closed without removing the flag, maybe we should reopen this. I'll update this line.